org.eclipse.dawnsci.nexus/src/org/eclipse/dawnsci/nexus/NexusUtils.java

```diff
@@ -719,4 +719,84 @@ private static boolean axesCompatible(int[] axesDims, int[] shapeAxes, int[] sha
 		}
 		return true;
 	}
+
+	/**
+	 * Compare object a as a scalar to String b by first converting b to a type
+	 * that matches a.
+	 *
+	 * @see Comparable#compareTo(Object)
+	 * @throws NumberFormatException
+	 *             if b can not be converted to the same type as a
+	 */
+	public static int compareScalarToString(Object a, String b)
+			throws NumberFormatException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.NumberFormatException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)